app/org/sagebionetworks/bridge/models/accounts/Phone.java

```diff
@@ -0,0 +1,72 @@
+package org.sagebionetworks.bridge.models.accounts;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import javax.persistence.Transient;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.i18n.phonenumbers.PhoneNumberUtil;
+import com.google.i18n.phonenumbers.PhoneNumberUtil.PhoneNumberFormat;
+import com.google.i18n.phonenumbers.Phonenumber.PhoneNumber;
+
+@Embeddable
+public class Phone {
+    
+    public static final boolean isValid(Phone phone) {
+        try {
+            PhoneNumber phoneNumber = PHONE_UTIL.parse(phone.getNumber(), phone.getRegionCode());
+            return PHONE_UTIL.isValidNumber(phoneNumber);
+        } catch (Exception e) {
+        }
+        return false;
```

Comment:
Unit tests for isValid?

Also, what cases throw an Exception? Do we want to catch all exceptions or just specific exceptions like NumberFormatException?